Configure

```diff
@@ -116,7 +116,8 @@ my $usage="Usage: Configure [no-<cipher> ...] [enable-<cipher> ...] [-Dxxx] [-lx
 # code, so we just tell compiler to be pedantic about everything
 # but 'long long' type.
 
-my $gcc_devteam_warn = "-DDEBUG_UNUSED"
+# These are common to all GCC compatible compilers
+our $devteam_common_warn = "-DDEBUG_UNUSED"
```

Comment:
I'm opposed to idea of omitting gcc from variable name. These *are* gcc-specific warnings and it's appropriate to denote them as such. But since we are here I'd be in favour of omitting devteam from variable name. The option grew beyond "for team to use" and it might as well be omitted. Why not say $gcc_strict_warnings?

Reply:
My original idea was to have some common ground that isn't necessarely originated in gcc...  but you're right, when push comes to shove, we do treat with the view that it originates from gcc, so...  So yeah, I'll change that name.